code/MediaWikiTest/MediaWikiTest/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace MediaWikiTest
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string language = "fr";
+            string text = string.Empty;
+            
+            text = fullSearch(string.Join("", args), language).Result;
+            
+            Console.WriteLine(text);
+            Console.ReadKey();
+        }
+
+        static async Task<string> fullSearch(string term, string language){
+            string text = await GetData(term, language);
+           
+                if(string.IsNullOrEmpty(text)){
+                    text = await GetData(term);
+                }
+
+                if(string.IsNullOrEmpty(text)){
+                    text = "No information found";
+             }
+                return text;
+        }
+
+        static async Task<string> GetOpenSearch(string term, string language)
```

Comment:
You can change this into ``` string language = "EN" ```

Reply:
I have already changed this in the Unity script.